.nvmrc

```diff
@@ -0,0 +1 @@
+8.11.2
```

Comment:
This will add another file to the project which means more maintenance. If going to add this, should be added to `npmignore` as this is for dev mode.

Reply:
Ah, well, the idea was that it would make dev easier for other folks (i.e., walk up to the repo and type `nvm use`), but I'm happy to remove it.  👍